yazi-core/src/tab/commands/escape.rs

```diff
@@ -1,29 +1,36 @@
 use bitflags::bitflags;
 use yazi_proxy::{AppProxy, ManagerProxy};
-use yazi_shared::{event::Cmd, render, render_and};
+use yazi_shared::{
+	event::{Cmd, Event, EventQuit},
+	render, render_and,
+};
 
 use crate::tab::Tab;
 
 bitflags! {
 	pub struct Opt: u8 {
-		const FIND   = 0b00001;
-		const VISUAL = 0b00010;
-		const SELECT = 0b00100;
-		const FILTER = 0b01000;
-		const SEARCH = 0b10000;
+		const FIND    = 0b0000001;
+		const VISUAL  = 0b0000010;
+		const SELECT  = 0b0000100;
+		const FILTER  = 0b0001000;
+		const SEARCH  = 0b0010000;
+		const QUIT    = 0b0100000;
+		const CASCADE = 0b1000000;
```

Comment:
This is to make it possible to explicitly invoke the default behavior (what happened previously when `opt` was empty)